test/test_meta.py

```diff
@@ -936,7 +936,7 @@ def __torch_function__(self, func, types, args=(), kwargs=None):
     aten._use_cudnn_ctc_loss.Tensor: {f32, f64},  # aten::_use_cudnn_ctc_loss.Tensor
     aten.cudnn_grid_sampler.default: {f16, f32, f64},  # aten::cudnn_grid_sampler
```

Comment:
@ngimel do cudnnSpatialTfSamplerForward and cudnnSpatialTfSamplerBackward support BFloat16?

Reply:
I don't know, you can try